### DISCUSSION TITLE
Folders/Files ignored but not in .ppignore

### DISCUSSION THREAD

**Discussion by restauffer** [photoprism/photoprism#4541] [Q&A]
My instance is ignoring all files/folders that start with the string "Bruce". I have looked at all .ppignore files and that string it not in them. However, I did have a .ppignore file in a sub-folder that did include that string previously. It was removed from that file some time past. I can see in the log where it ignores the files/folder starting with "Bruce".  How can I fix this?

Update: Doing some testing I have discovered that PP is ignoring any new file/folder that starts with a, A, b, or B. Most files/folders are there as it must have worked at some time and discovered all of them so only new ones are ignored.

**Comment by graciousgrey** [comment]
Did you restart your PhotoPrism instance after you have removed the string from all .ppignore files?

**Reply by restauffer** [reply]
Yes. I tried restarting PP but it did not help.

**Reply by restauffer** [reply]
Problem Solved:  In the top level .ppignore file I had this entry:
[AB]*

It should have been:
AB*
